### ISSUE TITLE
TypeScriptVersion.isRedirectable is not a function

### ISSUE THREAD

**Issue by ffflorian** [Microsoft/dtslint#163]
When running dtslint v0.4.1 with the command `dtslint types`, I get the following error:

```
definitelytyped_header_parser_1.TypeScriptVersion.isRedirectable is not a function
    at /home/travis/build/ffflorian/jsrsasign-types/node_modules/dtslint/bin/index.js:148:102
    at Generator.next (<anonymous>)
    at /home/travis/build/ffflorian/jsrsasign-types/node_modules/dtslint/bin/index.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/home/travis/build/ffflorian/jsrsasign-types/node_modules/dtslint/bin/index.js:4:12)
    at testTypesVersion (/home/travis/build/ffflorian/jsrsasign-types/node_modules/dtslint/bin/index.js:143:12)
    at /home/travis/build/ffflorian/jsrsasign-types/node_modules/dtslint/bin/index.js:128:19
    at Generator.next (<anonymous>)
    at fulfilled (/home/travis/build/ffflorian/jsrsasign-types/node_modules/dtslint/bin/index.js:5:58)
```

**Comment by ffflorian** [comment]
Pinging @andy-ms and @sandersn :slightly_smiling_face:

**Comment by sandersn** [comment]
1. What was the directory structure where you ran this command? That is, the working directory and child directories like `types`.
2. What was the exact command? I wasn't aware that `dtslint types` did anything. I thought it had to be `dtslint types "ansicolors"` (for example).

**Comment by ffflorian** [comment]
My initial bug report was a bit unclear, sorry.

I am creating types for a big library and, before creating a PR at DefinitelyTyped, I created my own repository to work on the types. To test the compatibility I installed `dtslint` and ran it with `dtslint types`, because `types` is my types directory which includes a `index.d.ts` file.

This is my directory structure:
```
.
├── LICENSE
├── node_modules
│   └── [...]
├── package.json
├── README.md
├── types
│   ├── index.d.ts
│   ├── jsrsasign.d.ts
│   ├── modules
│   ├── tsconfig.json
│   └── tslint.json
└── yarn.lock
```

I locally tried a few commands with different results:

```
$ npx dtslint
Error: ENOENT: no such file or directory, open '/[redacted]/jsrsasign-types/index.d.ts'

$ npx dtslint types
TypeError: definitelytyped_header_parser_1.TypeScriptVersion.isRedirectable is not a function
    at /[redacted]/jsrsasign-types/node_modules/dtslint/bin/index.js:148:102
    at Generator.next (<anonymous>)
    at /[redacted]/jsrsasign-types/node_modules/dtslint/bin/index.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/[redacted]/jsrsasign-types/node_modules/dtslint/bin/index.js:4:12)
    at testTypesVersion (/[redacted]/jsrsasign-types/node_modules/dtslint/bin/index.js:143:12)
    at /[redacted]/jsrsasign-types/node_modules/dtslint/bin/index.js:128:19
    at Generator.next (<anonymous>)
    at fulfilled (/[redacted]/jsrsasign-types/node_modules/dtslint/bin/index.js:5:58)

$ npx dtslint types/index.d.ts
Error: ENOTDIR: not a directory, open '/[redacted]/jsrsasign-types/types/index.d.ts/index.d.ts'
```

**Comment by ffflorian** [comment]
After the latest release v0.4.2, I get a bit further but in the end still the same error:

```
$ dtslint types
Installing to /github/workspace/node_modules/dtslint/typescript-installs/2.0...

Installed!
Installing to /github/workspace/node_modules/dtslint/typescript-installs/2.1...

Installed!
Installing to /github/workspace/node_modules/dtslint/typescript-installs/2.2...

Installed!
Installing to /github/workspace/node_modules/dtslint/typescript-installs/2.3...

Installed!
Installing to /github/workspace/node_modules/dtslint/typescript-installs/2.4...

Installed!
Installing to /github/workspace/node_modules/dtslint/typescript-installs/2.5...

Installed!
Installing to /github/workspace/node_modules/dtslint/typescript-installs/2.6...

Installed!
Installing to /github/workspace/node_modules/dtslint/typescript-installs/2.7...

Installed!
Installing to /github/workspace/node_modules/dtslint/typescript-installs/next...

Installed!
TypeError: definitelytyped_header_parser_1.TypeScriptVersion.isRedirectable is not a function
    at /github/workspace/node_modules/dtslint/bin/index.js:153:102
    at Generator.next (<anonymous>)
    at /github/workspace/node_modules/dtslint/bin/index.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/github/workspace/node_modules/dtslint/bin/index.js:4:12)
    at testTypesVersion (/github/workspace/node_modules/dtslint/bin/index.js:148:12)
    at /github/workspace/node_modules/dtslint/bin/index.js:133:19
    at Generator.next (<anonymous>)
    at fulfilled (/github/workspace/node_modules/dtslint/bin/index.js:5:58)
error Command failed with exit code 1.
```

**Comment by ffflorian** [comment]
@sandersn any update on this?

**Comment by ffflorian** [comment]
It was fixed with version 0.4.4 for me :tada:
